docs/application/native/get-started/iot/first-app.md

```diff
@@ -0,0 +1,292 @@
+# Create Your First Tizen IoT Native Service Application
+
+**Welcome to Tizen IoT Native Service Application development!**
+
+> [!NOTE]
+> Service applications are Tizen native applications, which run in the background with no graphical user interface.
+> These applications can be very useful for activities, such as getting sensor data in the background that needs to run periodically or continuously but does not require any user interaction.
+
+An IoT native service application can be created using the C language and run on Raspberry Pi. The application uses the native APIs, which provide various interfaces to the device hardware allowing you to take advantage of numerous capabilities tailored to run with limited device resources.
+
+You can study the following contents to help familiarize yourself with the Tizen [Native application development process](../../tutorials/process/app-dev-process.md) as well as using Tizen Studio and deploying the created application on Raspberry Pi. With the instructions, you can create and run a basic IoT native service application, which displays some text on the log with no user interaction:
+
+1. Before you get started with developing Tizen applications on Raspberry Pi, you must set up your development environment with the board.
+
+    For more information on setting up the development environment with the board, see the following:
+    - [Flashing Tizen Image](http://tizenschool.org/tutorial/191/contents/6)
+    - [Setting up Raspberry Pi](http://tizenschool.org/tutorial/191/contents/8)
+
+2. Download and install [Tizen Studio](../../../tizen-studio/index.md).
+
+    For more information on the installation process, see the [installation guide](../../../tizen-studio/setup/install-sdk.md).
+
+3. [Create an IoT native service project](#create) using Tizen Studio.
+
+    This step shows how to use a pre-designed template that creates all the basic files and folders required for your project.
+
+4. [Build the application](#build).
+
+    This step shows how to build your own application by validating and compiling your codes.
+
+5. [Run the application](#run).
+
+    This step shows how to run your application on Raspberry Pi.
+
+When you are developing a more complex application, you can take advantage of the [native tools included in Tizen Studio](../../../tizen-studio/native-tools/index.md) to ease the tasks of creating functionality and designing the application UI.
+
+<a name="create"></a>
+## Creating a Project
```

Comment:
@ey928-lee, please chnage all the heading to sentence case:

Line 38: ## Create project
Line 112: ### Manage application configuration
Line 141: ### Understand source code
Line 174: ### Add log
Line 206: ## Build your application
Line 247: ## Run your application
Line 252: ### Run on Raspberry Pi

@mijin85cho, please verify

Reply:
@surbhig12 I changed all the heading into your suggestions.
@mijin85cho Please verify :)